x/concentrated-liquidity/range_test.go

```diff
@@ -266,6 +349,19 @@ func (s *KeeperTestSuite) executeRandomizedSwap(pool types.ConcentratedPoolExten
 
 	swapOutCoin := sdk.NewCoin(swapOutDenom, baseSwapOutAmount)
 
+	// If the swap we're about to execute will not generate enough input, we skip the swap.
+	if swapOutDenom == pool.GetToken1() {
+		pool, err := s.clk.GetPoolById(s.Ctx, pool.GetId())
+		s.Require().NoError(err)
+
+		poolSpotPrice := pool.GetCurrentSqrtPrice().Power(2)
+		minSwapOutAmount := poolSpotPrice.Mul(sdk.SmallestDec()).TruncateInt()
+		poolBalances := s.App.BankKeeper.GetAllBalances(s.Ctx, pool.GetAddress())
+		if poolBalances.AmountOf(swapOutDenom).LTE(minSwapOutAmount) {
+			return sdk.Coin{}, sdk.Coin{}
+		}
+	}
```

Comment:
Why is this only done for swapOutDenom of token 1?

Reply:
This specific methodology (multiplying by sqrt price, which is asset1 / asset0) only applies for asset1, but I agree this check could be added in the other direction as well